### PR TITLE
Fix small logos

### LINKED ISSUE #12441: Someone zapped our logo with a shrink ray
The Metabase logo on the setup page looks smaller than it should:

Screen Shot 2020-04-30 at 3 21 15 PM

### PR DESCRIPTION
**Pull request by kdoh**
So this one is a little weird but I *think* the reason certain logos started getting smaller without any particularly obvious reason at the site of their use was the whole "DefaultIcon" switch to help plugin-ify things. I think the size prop ended up taking precedence due to the use of `||` and because it had a default value via `defaultProps.`

The addition of the size prop itself was probably never a great idea. We don't use it anywhere in the code and the logo also isn't quite square but regardless the idea of having two different size props is just kind of bad component api design. Mix that in with how width and size interact with the SVG viewBox and suddenly you end up with a lot of confusion about what the heck the size of the thing is going to be. Happily this is a case of the saner component API also fixing the bug.

I still don't love that the primary way we seem to tweak the size is via "height" based on a look through the codebase, but that's small potatoes in the grand scheme of things.

@paulrosenzweig / @tlrobinson would this introduce any potential issues with white label logos? I believe we just swap the entire component out right so internal prop changes shouldn't matter?

Closes #12441

### CODE REVIEW

File: frontend/src/metabase/components/LogoIcon.jsx [hunk 1]
```diff
@@ -5,25 +5,20 @@ import cx from "classnames";
...
-    const { dark, height, width, size } = this.props;
+    const { dark, height, width } = this.props;
     return (
       <svg
         className={cx("Icon", { "text-brand": !dark }, { "text-white": dark })}
         viewBox="0 0 66 85"
-        width={width || size}
-        height={height || size}
+        width={width}
+        height={height}
```
Comment (paulrosenzweig) [inline frontend/src/metabase/components/LogoIcon.jsx:21]: Are there any places we don't set height? Is it worth having a default here or does the viewbox dimension take effect if unset?

**Review by paulrosenzweig — APPROVED** [review]
Nice fix! I think you're right that this shouldn't affect anything if a plugin overrides the logo.

File: frontend/src/metabase/components/LogoIcon.jsx (hunk 1 above)
Comment (kdoh) [reply frontend/src/metabase/components/LogoIcon.jsx:21]: Good question. I'll double check that everything still works ok if you don't specify any props.

File: frontend/src/metabase/components/LogoIcon.jsx [hunk 2]
```diff
@@ -37,6 +35,7 @@ class DefaultLogoIcon extends Component {
 }
 
 export default function LogoIcon(props) {
+  console.log("logo props?", props);
```
Comment (flamber) [inline frontend/src/metabase/components/LogoIcon.jsx:38]:
```suggestion
```
Was that on purpose?

Comment (kdoh) [reply frontend/src/metabase/components/LogoIcon.jsx:38]: Super not! Thanks for the catch.

**Pull request merged**